Meta/lint-keymaps.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python3
+
+import json
+import os
+import sys
+
+
+PERMITTED_MAPS = ['map', 'shift_map', 'alt_map', 'altgr_map', 'shift_altgr_map']
+REQUIRED_MAPS = ['map', 'shift_map', 'alt_map']
+# See Userland/Libraries/LibKeyboard/CharacterMapFile.cpp
+# and Userland/Libraries/LibKeyboard/CharacterMap.cpp.
+GOOD_MAP_LENGTHS = {90, 128}
+
+
+def report(filename, problem):
+    print('{}: {}'.format(filename, problem))
+
+
+def validate_single_map(filename, mapname, values):
+    all_good = True
+
+    if not isinstance(values, list):
+        report(filename, '"{}" is not a list'.format(mapname))
```

Comment:
Let's use JSON terminology here:

```suggestion
        report(filename, '"{}" is not an array'.format(mapname))
```